### PR TITLE
Azure support multiline secret values

### LINKED ISSUE #780: Enhance Azure Key Vault key import to mimic AWS
Importing from AKV on a secret by secret basis takes around 7minutes (for 10K keys). In AWS, using the multiple keys per secret mechanism, it takes around 6seconds.

We'd like the same 'batching' made available to keys stored in Azure Key Vault as per AWS Secrets Manager.  This will significantly impact downtime and help reduce the impact of upgrades etc. across our platform.

### PR DESCRIPTION
**Pull request by usmansaleem**
## PR Description
Support Azure multiline secret values in bulk mode.

## Fixed Issue(s)

Fixes #780 

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

## Testing

- [x] I thought about testing these changes in a realistic/non-local environment.

### CODE REVIEW

File: core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java [hunk 1]
```diff
@@ -273,6 +273,11 @@ protected ArtifactSignerProvider createArtifactSignerProvider(
 
           if (azureKeyVaultParameters.isAzureKeyVaultEnabled()) {
             LOG.info("Bulk loading keys from Azure key vault ... ");
+            /*
+             Note: Azure supports 25K bytes per secret. https://learn.microsoft.com/en-us/azure/key-vault/secrets/about-secrets
+             Each raw bls private key in hex format is approximately 100 bytes. We should store about 200 or fewer
+             `\n` delimited keys per secret.
+            */
```
Comment (siladu) [inline core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java:280]: 👍

**Review by siladu — COMMENTED** [review]
Missing changelog :)

Wow, this is a super simple change following the AWS one and merging signers lib in, nice!

Are these features going to be documented anywhere? Feels like we should mention the size restrictions there too.

File: core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java (hunk 1 above)
Comment (usmansaleem) [reply core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java:280]: Thats the reason I added this comment, this is meant to be updated at https://docs.web3signer.consensys.net/how-to/use-signing-keys#bulk-load-consensus-layer-keys

**Review by siladu — APPROVED** [review]
LGTM

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/bulkloading/AzureKeyVaultAcceptanceTest.java [hunk 2]
```diff
@@ -43,16 +57,21 @@ public class AzureKeyVaultAcceptanceTest extends AcceptanceTestBase {
...
     Assumptions.assumeTrue(CLIENT_SECRET != null, "Set AZURE_CLIENT_SECRET environment variable");
     Assumptions.assumeTrue(TENANT_ID != null, "Set AZURE_TENANT_ID environment variable");
     Assumptions.assumeTrue(VAULT_NAME != null, "Set AZURE_KEY_VAULT_NAME environment variable");
+
+    createAndFindAzureMultilineKeysIfNotExist();
   }
 
   @Test
-  void ensureSecretsInKeyVaultAreLoadedAndReportedViaPublicKeysApi() {
+  void azureSecretsWithValidAndInvalidKeysWithoutTags() {
```
Comment (siladu) [inline acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/bulkloading/AzureKeyVaultAcceptanceTest.java:74]: This name seems less clear than the former one to me

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/bulkloading/AzureKeyVaultAcceptanceTest.java [hunk 3]
```diff
@@ -162,7 +191,54 @@ void envVarsAreUsedToDefaultAzureParams() {
...
     startSigner(configBuilder.build());
 
-    final Response response = signer.callApiPublicKeys(KeyType.BLS);
-    response.then().statusCode(200).contentType(ContentType.JSON).body("", contains(EXPECTED_KEY));
+    final List<String> publicKeys =
+        multilineKeys.stream()
+            .map(BLSKeyPair::getPublicKey)
+            .map(BLSPublicKey::toHexString)
+            .collect(Collectors.toList());
+    publicKeys.add(EXPECTED_KEY);
```
Comment (siladu) [inline acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/bulkloading/AzureKeyVaultAcceptanceTest.java:199]: Seems like we've replaced the more common single key scenario with the less common multiKey scenario. 
These are separate tests in AWS, feel like that's a better approach, even if the current implementation is equivalent to a "list of one".

**Review by siladu — APPROVED** [review]
LGTM

**Pull request merged**